src/main/java/uk/gov/justice/hmpps/prison/api/resource/ScheduleResource.java

```diff
@@ -74,13 +80,17 @@ public List<PrisonerSchedule> getEventsByLocationId(@PathVariable("agencyId") @P
+    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH"})
     @GetMapping("/{agencyId}/locations/{locationId}/usage/{usage}")
     @SlowReportQuery
-    public List<PrisonerSchedule> getLocationEvents(@PathVariable("agencyId") @Parameter(description = "The prison.", required = true) final String agencyId, @PathVariable("locationId") @Parameter(description = "The location id where event is held.", required = true) final Long locationId, @PathVariable("usage") @Parameter(description = "The locationUsage code from the location object - one of the INTERNAL_LOCATION_USAGE reference codes.", required = true) final String usage,
-                                                    @RequestParam(value = "date", required = false) @DateTimeFormat(iso = ISO.DATE) @Parameter(description = "Date of whereabouts list, default today") final LocalDate date, @RequestParam(value = "timeSlot", required = false) @Parameter(description = "AM, PM or ED") final TimeSlot timeSlot, @RequestHeader(value = "Sort-Fields", required = false) @Parameter(description = "Comma separated list of one or more of the following fields - <b>cellLocation or lastName</b>") final String sortFields,
+    public List<PrisonerSchedule> getLocationEvents(@PathVariable("agencyId") @Parameter(description = "The prison.", required = true) final String agencyId,
+                                                    @PathVariable("locationId") @Parameter(description = "The location id where event is held.", required = true) final Long locationId,
+                                                    @PathVariable("usage") @Parameter(description = "The locationUsage code from the location object - one of the INTERNAL_LOCATION_USAGE reference codes.", required = true) final String usage,
+                                                    @RequestParam(value = "date", required = false) @DateTimeFormat(iso = ISO.DATE) @Parameter(description = "Date of whereabouts list, default today") final LocalDate date,
+                                                    @RequestParam(value = "timeSlot", required = false) @Parameter(description = "AM, PM or ED") final TimeSlot timeSlot,
+                                                    @RequestHeader(value = "Sort-Fields", required = false) @Parameter(description = "Comma separated list of one or more of the following fields - <b>cellLocation or lastName</b>") final String sortFields,
                                                     @RequestHeader(value = "Sort-Order", defaultValue = "ASC", required = false) @Parameter(description = "Sort order (ASC or DESC) - defaults to ASC.") final Order sortOrder) {
-
-        return schedulesService.getLocationEvents(agencyId, locationId, usage, date, timeSlot, sortFields, sortOrder);
+        return schedulesService.getLocationEvents(locationId, usage, date, timeSlot, sortFields, sortOrder);
```

Comment:
agencyId unused

Reply:
It wasn't used for ` @VerifyAgencyAccess` in some lower level service or whatever?

Reply:
It was, but not now the annotation has moved.